packages/router/src/lib/helpers/shared.ts

```diff
@@ -42,15 +70,37 @@ export const calculateGasFeeInReceivingToken = async (
   // if chaindIdsForGasFee includes only sendingChainId, calculate gas fee for fulfill transactions
   // if chaindIdsForGasFee includes only receivingChainId, calculate gas fee for prepare transactions
 
+  // NOTE: to handle optimism gas fees before oracle is deployed, use mainnet
+  // oracle token pricing and optimism gas price
+  const tokenPricingSendingChain = sendingChainId === 10 ? 1 : sendingChainId;
```

Comment:
Can we just have a hardcoded const at the top for all NO_ORACLE_CHAINS? would feel slightly better

Reply:
sure